pkgs/by-name/al/altdns/package.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication {
+  pname = "altdns";
+  version = "unstable-2021-09-09";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "infosec-au";
+    repo = "altdns";
+    rev = "8c1de0fa8365153832bb58d74475caa15d2d077a";
+    hash = "sha256-ElY6AZ7IBnOh7sRWNSQNmq7AYGlnjvYRn8/U+29BwWA=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+    wheel
```

Usually not needed for plain Python modules/tools.

```suggestion
    wheel
```